### ISSUE TITLE
tkg_vault_ip meaning

### ISSUE THREAD

**Issue by lkalaivanan** [tanzuformers/terraform-tkg-tkgclustersetup#1]
What is tkg_vault_ip  means here and how to configure one ?

**Comment by manuelcoppotelli** [comment]
Hi @lkalaivanan 

it is the IP address of a Hashicorp Vault server.

(cc @linotelera )

**Comment by linotelera** [comment]
Yes, it is! We use a vault server provided by terraform-helm-tkgmgmtsetup module to store all secrets

**Comment by lkalaivanan** [comment]
@manuelcoppotelli and @linotelera 

Thanks will try this
